### ISSUE TITLE
panel data

### ISSUE THREAD

**Issue by lannistercaer** [cdt15/lingam#31]
hello

can I appy var-lingam model with panel data? If yes, is there any tutorial or document for me to learn how to do it with panel data?

**Comment by sshimizu2006** [comment]
Hi, 

This example might help you: 
https://github.com/cdt15/lingam/blob/master/examples/LongitudinalLiNGAM.ipynb

The longitudinal LiNGAM model was proposed in this paper: https://189568f5-a-62cb3a1a-s-sites.googlegroups.com/site/sshimizu06/Kadowaki13MLSP.pdf?attachauth=ANoY7crn2js1f4SCYtfKU983U-7DN6JLQH9_phos98lyi5dhYNQSf2eNbz_xQIpHbxN3QlvM5B9W7_zgScHzzOk3bvZQk6RIukagdE4uWL9tWVKa-uFHUA55RyHd0nxlC3xEle0K5Tho8DgxUdLu9cYAQj9YPV1J2r5jpakZGmUsL5I_1WoX3-0LfmKpnxfYXldAlRA5Of46EyKCGpvui9ligQHsfjdLLA%3D%3D&attredirects=0

**Comment by lannistercaer** [comment]
hello， sshimizu2006

Very thanks to your help! So, longitudinal LiNGAM is suitable for panel data instead of var-lingam, is that right?

**Comment by sshimizu2006** [comment]
Yes if panel data means collecting data from multiple persons over time.

**Comment by lannistercaer** [comment]
hello, sshimizu2006

sorry for asking again.

https://onlinelibrary.wiley.com/doi/full/10.1111/j.1468-0084.2012.00710.x?saml_referrer

As in this paper , In the section 'IV. Microeconomic application: firm growth', It seems that the panel dataset has been analyzed by the VAR-LiNGAM model. Am I wrong with that?

**Comment by sshimizu2006** [comment]
Hi, 

In the section, firms are pooled (the 8th line below Eq. (2)). So, the dataset is a kind of one "person" over time.

Shohei

**Comment by lannistercaer** [comment]
Hi, sshimizu2006

thank you for your patience to answer my question!
